src/ThreeEditor/js/Menubar.File.js

```diff
@@ -56,7 +55,7 @@ function MenubarFile(editor) {
 
 	option = new UIRow();
 	option.setClass('option');
-	option.setTextContent(strings.getKey('menubar/file/import'));
+	option.setTextContent('Open');
```

Comment:
If we have export, the this one should be called import
```suggestion
	option.setTextContent('Import');
```

Reply:
But that action is erasing the current state in the editor. I think it is better to rename `Export` to `Save`.

Reply:
Then what about renaming `Export` to `Clean and Save` ?